formatter.go

```diff
@@ -14,33 +15,62 @@ import (
 // the write entry
 type Formatter interface {
```

Comment:
SetFields is defined in the MutableLogger interface. Should not SetFieldMappings be part of MutableLogger as well?

Reply:
Fixed, also renamed it to `SetFieldMapping`

Reply:
fixed